README.md

```diff
@@ -591,16 +600,34 @@ $ newman run file-upload.postman_collection.json
 
 ## Using Newman with the Postman API
 
-1 [Generate an API key](https://app.getpostman.com/dashboard/integrations)<br/>
-2 Fetch a list of your collections from: `https://api.getpostman.com/collections?apikey=$apiKey`<br/>
-3 Get the collection link via it's `uid`: `https://api.getpostman.com/collections/$uid?apikey=$apiKey`<br/>
-4 Obtain the environment URI from: `https://api.getpostman.com/environments?apikey=$apiKey`<br/>
-5 Using the collection and environment URIs acquired in steps 3 and 4, run the collection as follows:
-```console
-$ newman run "https://api.getpostman.com/collections/$uid?apikey=$apiKey" \
-    --environment "https://api.getpostman.com/environments/$uid?apikey=$apiKey"
+Newman supports the access of Postman-Cloud resources by its ID/UID/URL. This is done using the Postman API Key.
+
+Click [here](https://app.getpostman.com/dashboard/integrations) to generate this key.
+
+### `newman login`
+This is an interactive command to store the Postman API Key to be used to communicate with Postman. The access to the API Key can be restricted with a passkey.
+
+``` console
+$ newman login
+```
+
+Following this initial process, Postman-Cloud resources can be accessed directly from its ID/UID/URL as follows.
+
+``` console
+$ newman run $collection_id -e $environment_id
 ```
 
+This virtual session only comes to an end when the following command is used.
+
+### `newman logout`
+This command deletes the stored authentication details.
+
+``` console
+$ newman logout
+```
+
+To know more about newman-login and its advanced features, click [here](https://github.com/narayanpai1/newman/wiki/Using-Newman-with-the-Postman-API).
```

Comment:
After the wiki is added to Newman's repository, I will change this link to point to that wiki.

Reply:
https://github.com/postmanlabs/newman/wiki/Using-Newman-with-the-Postman-API